concent_api/core/views.py

```diff
@@ -1241,6 +1378,240 @@ def store_message_and_message_status(
         receive_message_status.save()
 
 
+def store_subtask(
+    task_id:                        str,
+    subtask_id:                     str,
+    provider_public_key:            bytes,
+    requestor_public_key:           bytes,
+    state:                          Enum,
```

Comment:
:+1:

Reply:
You mean `mypy`? Or is it really a pylint error? Does pylint look at types?

Anyway, a quick search shows that mypy fully supports enums so I don't see why this would not work. It's not a priority but something we should look into when there's a bit more time.

Reply:
:+1: Actually the error was in different line, and mypy thought a wrong type is passed to another function, so it is working and I will fix it.